<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CropImage</class>
 <widget class="QDialog" name="CropImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crop Image</string>
  </property>
  <widget class="MouseTrack_QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MousePosition">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>740</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>X = 0, Y = 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_RgbValue">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>740</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>RGB: (0,0,0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_Crop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>730</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_Top">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>10</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Left">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>100</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Right">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>40</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Bottom">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>70</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Top">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Right">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Bottom">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Left">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X = 0, Y = 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_ImgExample">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>730</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseTrack_QLabel</class>
   <extends>QLabel</extends>
   <header location="global">mousetrack_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
